lib/utils.js

```diff
@@ -359,9 +356,9 @@ api.validateCapabilityChain = ({capability, capabilityChain}) => {
  *
  * @param {Object} capability - the JSON-LD document for the object capability,
  *          compacted to the security v2 context.
- * @param {Object} proof - the capability delegation proof to use if the
- *          capability's chain is being verified without invoking the
- *          capability.
+ * @param {Boolean} [excludeGivenCapability=false] - `true` to exclude
```

Comment:
I don't know if `Boolean` or `boolean` is right, but here's a suggestion if `boolean` is right:

```suggestion
 * @param {boolean} [excludeGivenCapability=false] - `true` to exclude
```